specification/azurefleet/AzureFleet.Management/common.tsp

```diff
@@ -1,19 +1,44 @@
 import "@typespec/http";
 import "@typespec/rest";
 import "@typespec/versioning";
-import "@azure-tools/typespec-autorest";
 import "@azure-tools/typespec-azure-core";
 import "@azure-tools/typespec-azure-resource-manager";
 
 using TypeSpec.Http;
 using TypeSpec.Rest;
 using TypeSpec.Versioning;
-using Autorest;
 using Azure.ResourceManager;
 using OpenAPI;
 
 namespace Microsoft.AzureFleet;
 
 @doc("ApiError for Fleet")
-@useRef("../../../../../compute/resource-manager/Microsoft.Compute/common-types/v1/common.json#/definitions/ApiError")
-model ApiError {}
+model ApiError is ApiErrorBase {
+  @extension("x-ms-identifiers", ["message", "target"])
+  @doc("The Api error details")
+  details?: Array<ApiErrorBase>;
+
+  @doc("The Api inner error")
+  innererror?: InnerError;
+}
+
+@doc("Api error base.")
+model ApiErrorBase {
+  @doc("The error code.")
+  code?: string;
+
+  @doc("The target of the particular error.")
+  target?: string;
+
+  @doc("The error message.")
+  message?: string;
+}
+
+@doc("Inner error details.")
+model InnerError {
+  @doc("The exception type.")
+  exceptionType?: string;
+
+  @doc("The internal error message or exception dump.")
+  errorDetail?: string;
```

Comment:
According to definition it should be `errordetail` and `exceptiontype`, https://github.com/MaryGao/azure-rest-api-specs/blob/51d8dc5c05cdaeb436b41dc9ae7336ebf7b1e121/specification/compute/resource-manager/models.tsp#L3298-L3308.

Reply:
In the server definition it is camel case: https://msazure.visualstudio.com/One/_git/Compute-CPlat-Core?path=/src/Shared/Lib/ResourceProvider.Service/API/RpcEntityModel.cs&version=GBmaster&line=128&lineEnd=144&lineStartColumn=9&lineEndColumn=38&lineStyle=plain&_a=contents

Reply:
cc @grizzlytheodore to confirm if this was missed in the swagger.

Reply:
Discussed offline and the casing should be camel case.

Reply:
@SahilArora92 In current Compute Swagger, it's not camel case, may I know the reason?
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/compute/resource-manager/Microsoft.Compute/common-types/v1/common.json#L59-L66

Reply:
They have it set up wrong. I confirmed from the compute spec release manager. 
![image](https://github.com/Azure/azure-rest-api-specs/assets/7030203/5dfc0c63-5e75-4584-88c3-dd8e522ea479)